### PR TITLE
UserId instead of UUID in Config struct

### PR DESCRIPTION
**Pull request by kleewho**

### CODE REVIEW

File: config.go [hunk 1]
```diff
@@ -58,10 +64,9 @@ func NewDemoConfig() *Config {
 
 }
 
-// NewConfig initiates the config with default values.
-func NewConfig(uuid string) *Config {
+func NewConfigWithUserId(userId UserId) *Config {
 	c := Config{
-		UUID:                          uuid,
```
Comment (phairow) [inline config.go:64]: Is there a reason why we don't rename `UUID` to `UserId` within the Config?

File: config.go [hunk 2]
```diff
@@ -10,17 +10,23 @@ const (
...
-	UUID                          string             // UUID to be used as a device identifier.
+	PublishKey   string // PublishKey you can get it from admin panel (only required if publishing).
+	SubscribeKey string // SubscribeKey you can get it from admin panel.
+	SecretKey    string // SecretKey (only required for modifying/revealing access permissions).
+	AuthKey      string // AuthKey If Access Manager is utilized, client will use this AuthKey in all restricted requests.
+	Origin       string // Custom Origin if needed
+
+	// UUID to be used as a device identifier.
+	//
+	//Deprecated: please use SetUserId/GetUserId
```
Comment (phairow) [inline config.go:28]: It is nice to see the deprecation comments

**Review by phairow — APPROVED** [review]
LGTM! I added a comment about UUID but it seems to be internal so I'm not really concerned about it.

File: config.go (hunk 1 above)
Comment (kleewho) [reply config.go:64]: Yes, indeed. The reason is that we don't want to break the code, yet. We might need to give a warning at least that sooner or later `UUID` will be removed.

**Review by techwritermat — APPROVED** [review]

**Pull request merged**